docs/mast/mast.rst

```diff
@@ -799,13 +812,33 @@ ccd a target pixel file will be produced for each one.
                   2  APERTURE      1 ImageHDU        80   (5, 5)   int32
 
 
-The `~astroquery.mast.TesscutClass.download_cutouts` function takes a coordinate or object name
-(such as "M104" or "TIC 32449963") and cutout size (in pixels or an angular quantity) and
-downloads the cutout target pixel file(s).
+Requesting a cutout by moving_target accesses the
+`MAST Moving Target TESScut API <https://mast.stsci.edu/tesscut/docs/getting_started.html#moving-target-cutouts>`__
+and returns a target pixel file, with format described
+`here <https://astrocut.readthedocs.io/en/latest/astrocut/file_formats.html#path-focused-target-pixel-files>`__.
+The moving_target is an optional bool argument where `True` signifies that the accompanying `objectname` input is the object name or ID understood by the
```

Comment:
The raw log files of the documentation builds are available from the URL `https://readthedocs.org/api/v2/build/{id}.txt` where `{id}` is an index that gets incremented every time a new build in started. Once the index values for the last successful and first failing build have been looked up it is possible to use the following shell function to quickly see what caused the failure:
```shell
check_readthedocs() {
  diff <(curl -s https://readthedocs.org/api/v2/build/$1.txt | sort) <(curl -s https://readthedocs.org/api/v2/build/$2.txt | sort) | grep WARNING;
}
```
In this case the relevant build indices are 15919123 and 15919240, and the output of `check_readthedocs 15919123 15919240` is
```
> /home/docs/checkouts/readthedocs.org/user_builds/astroquery/checkouts/2121/docs/mast/mast.rst:815: WARNING: py:obj reference target not found: objectname
```
This points out the problem and its solution quite clearly:
```suggestion
The moving_target is an optional bool argument where `True` signifies that the accompanying ``objectname`` input is the object name or ID understood by the
```